packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -0,0 +1,233 @@
+// @ts-check
+import { E } from '@endo/eventual-send';
+import {
+  assertPathSegment,
+  makeStorageNodeChild,
+} from '@agoric/internal/src/lib-chainStorage.js';
+import { reserveThenDeposit } from './utils.js';
+
+const trace = (...args) => console.log('GovReplaceCommiteeAndCharter', ...args);
+
+const { values } = Object;
+
+/** @type {<X, Y>(xs: X[], ys: Y[]) => [X, Y][]} */
+const zip = (xs, ys) => xs.map((x, i) => [x, ys[i]]);
+
+/** @type {(name: string) => string} */
+const sanitizePathSegment = name => {
+  const candidate = name.replace(/[ ,]/g, '_');
+  assertPathSegment(candidate);
+  return candidate;
+};
+
+const handlehighPrioritySendersList = async (
+  { consume: { highPrioritySendersManager: highPrioritySendersManagerP } },
+  { options: { highPrioritySendersConfig } },
+) => {
+  const HIGH_PRIORITY_SENDERS_NAMESPACE = 'economicCommittee';
+  const highPrioritySendersManager = await highPrioritySendersManagerP;
+
+  if (!highPrioritySendersManager) {
+    throw Error('highPrioritySendersManager is not defined');
+  }
```

Comment:
```suggestion
  highPrioritySendersManager || Fail`highPrioritySendersManager is not defined`;
```

If this isn't in [Coding-Style](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style), it should be.